README.md

```diff
@@ -26,7 +26,12 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+ def offer_rose(person)
+   puts ' Would you take this rose and help out an old beggar, #{person}?'
+   end
```

Comment:
You have an extra `end` in here. The only thing you need to close is the `offer_rose1` method.